lms/resources/_js_config/__init__.py

```diff
@@ -491,14 +491,13 @@ def _blackboard_sync_api(self):
 
     def _sync_api(self):
```

Comment:
Maybe `_get_sync_api` or something? This sounds like a verb not a noun

Reply:
Looks like we might want a:

```python
if not self._context.is_group_launch:
    return None
```

At the top here?

It would also be nice if this was before the functions it calls as the way you read through the file is kind of 1, 3, 2 at the moment.